README.md

```diff
@@ -105,4 +105,9 @@ Using this configuration, the health endpoint provides the following result:
         }
     }
 }
-```
\ No newline at end of file
+```
+
+## SBOM
+
+To generate a software bill of materials (SBOM), execute the gradle task cyclonedxBom.
```

Comment:
Please wrap the task name in `` ` `` characters.

Example:
...the gradle task `cyclonedxBom`.